src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+  const message = document.createElement('div');
+
+  message.className = 'message';
+
+  logo.addEventListener('click', () => {
+    const text = 'Promise was resolved!';
+
+    message.append(text);
+  });
+  resolve(message);
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  const message = document.createElement('div');
+
+  message.className = 'message';
+  message.classList.add('error-message');
```

Comment:
The `error-message` class is added to the message for `promise2`, which is fine if the task requires a specific styling for error messages. Ensure that this aligns with the task requirements.